src/background/service/APIService.js

```diff
@@ -639,13 +643,89 @@ class APIService {
         params.method = TRANSACTION_TYPE.ReclaimEscrow
         return this.submitTxBody(params, tw)
     }
+    /**
+     * withdraw amount when set allowance
+     * @param {*} params 
+     * @returns 
+     */ 
+    setWithdraw = async (params) => {
+        const CONSENSUS_RT_ID = oasis.misc.fromHex(params.toAddress)
+        const consensusWrapper = new oasisRT.consensusAccounts.Wrapper(CONSENSUS_RT_ID);
+        const withdrawWrapper = consensusWrapper.callWithdraw()
+        return this.submitRuntimeBody(params, withdrawWrapper)
+    }
+    /**
+     * set deposit
+     * @param {*} params 
+     */
+    setDeposit= (params)=>{
```

Comment:
@lvshaoping007 Would these be more correct function names?
```ts
APIService.setDeposit   // setAllowanceAndDepositToConsensusAccount?
  oasis.staking.allowWrapper()
    amount = allowanceDifference
  APIService.submitTxBody onBroadcastEnd
  APIService.onBroadcastEnd    // depositToConsensusAccount?
    APIService.submitRuntimeBody   // submitDepositToConsensusAccountTx?
      APIService.buildRuntimeTxBody   // buildDepositToConsensusAccountTx?
        oasisRT.consensusAccounts.Wrapper.callDeposit()
          amount = DEPOSIT_AMOUNT
      submitTx
      APIService.checkRuntimeTxStatus   // createNotificationAfterRuntimeTxSucceeds?
        extension.notifications.create
```

Reply:
ok, i will fix this in next PR ， but  what does this means , should this be fixed ? 
```
oasis.staking.allowWrapper()
    amount = allowanceDifference
    
oasisRT.consensusAccounts.Wrapper.callDeposit()
          amount = DEPOSIT_AMOUNT
      submitTx
```

Reply:
I think those lines are included for context, and the names used there are fine

Reply:
ok